content/en/docs/components/katib/experiment.md

```diff
@@ -436,6 +437,21 @@ Katib supports the following algorithm settings:
   </table>
 </div>
 
+<a id="sobol"></a>
+
+#### Sobol's Quasirandom Sequence
+
+The algorithm name in Katib is `sobol`.
+
+Katib uses the [Goptuna](https://github.com/c-bata/goptuna) optimization
+framework for its Sobol's quasirandom search.
+
+The [Sobol's quasirandom sequence](https://dl.acm.org/doi/10.1145/641876.641879)
+is a low-discrepancy sequence. And it is known that Sobol's quasirandom sequence can
+provide better uniformity properties.
+
+Currently, Katib does not support no algorithm settings for Sobol.
```

Comment:
I think we can remove this notification similar to Grid: https://www.kubeflow.org/docs/components/katib/experiment/#grid-search.

```suggestion
```